src/twisted/web/test/test_flatten.py

```diff
@@ -272,41 +298,50 @@ def test_commentEscaping(self) -> Deferred[List[bytes]]:
         SGML comment syntax is complicated and hard to use. This rule is more
         restrictive, and more compatible:
 
-        Comments start with <!-- and end with --> and never contain -- or >.
+        Comments start with <!-- and end with --> and HTML comments never contain -- or >,
+        whereas Outlook conditional comments contain their logic within brackets 
+        that follow this format <!-- [if foo]>
 
         Also by XML syntax, a comment may not end with '-'.
 
         @see: U{http://www.w3.org/TR/REC-xml/#sec-comments}
         """
 
-        def verifyComment(c: bytes) -> None:
+        def verifyComment(z: bytes) -> None:
```

Comment:
because `c` was used as a variable name, whenever I tested the output with a breakpoint and pressed `c` to check the content of the variable "continue" was activated. Perhaps that is why I mishandled my revision of the test cases? Not sure, but this now works as expected.

Reply:
why not rename it to "comment" or "targetedComment" or "expectedComment" or something that will explain what this variable is all about.

I am not a mathematician, and I don't like single letter variables :)